frontend/src/components/network-presets/network-presets.tsx

```diff
@@ -154,7 +154,7 @@ export function NetworkPresets({
           Show test networks
         </ButtonUnstyled>
       )}
-      {myNetworks.length > 0 && <Header>My Networks</Header>}
+      {myNetworks.length > 0 && <Title>My Networks</Title>}
```

Comment:
react doesn't work that way. if it sees a `0`, it will render it. we don't want zeros floating around on the UI, so this needs to be false, undefined or null. 

fyi: https://www.designcise.com/web/tutorial/why-is-react-showing-0-when-conditionally-rendering-a-component